gm4_bookshelf_nametags/data/gm4_bookshelf_nametags/functions/evaluate/motion.mcfunction

```diff
@@ -0,0 +1,25 @@
+# check player motion
+# @s = player not in spectator mode
+# at unspecified
+# run from evaluate/run
+
+# store player motion using a marker to avoid reading player NBT
+tag @s add gm4_bookshelf_nametags_target
+execute summon marker run function gm4_bookshelf_nametags:evaluate/get_pos
+tag @s remove gm4_bookshelf_nametags_target
+
+# round down to lowest integer, if all are 0 consider player to not be moving
+execute store result score $Mot_0 gm4_bookshelf_nametags_data run data get storage gm4_bookshelf_nametags:temp Motion[0]
```

Comment:
Fake players should be in `snake_case` and not shortened unless absolutely required; furthermore, translating 0, 1, and 2 to x, y, and z would make this easier to understand i.e. `$motion_x` instead of `$Mot_0`.